From c38256621d4dffbbc0c19737d724724f04b0df9a Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Thu, 16 May 2019 15:17:19 +0100
Subject: [PATCH] staging: vcsm-cma: Drop logging level on messages in
 vc_sm_release_resource

They weren't errors but were logged as such.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/staging/vc04_services/vc-sm-cma/vc_sm.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

--- a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
+++ b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
@@ -301,14 +301,13 @@ static void vc_sm_release_resource(struc
 
 	if (buffer->vc_handle) {
 		/* We've sent the unmap request but not had the response. */
-		pr_err("[%s]: Waiting for VPU unmap response on %p\n",
-		       __func__, buffer);
+		pr_debug("[%s]: Waiting for VPU unmap response on %p\n",
+			 __func__, buffer);
 		goto defer;
 	}
 	if (buffer->in_use) {
 		/* dmabuf still in use - we await the release */
-		pr_err("[%s]: buffer %p is still in use\n",
-		       __func__, buffer);
+		pr_debug("[%s]: buffer %p is still in use\n", __func__, buffer);
 		goto defer;
 	}
 
